appservice/src/connectToGitHub.ts

```diff
@@ -199,7 +201,7 @@ async function getGitHubReposQuickPicks(cache: ICachedQuickPicks<gitHubReposData
         }
     } while (requestOptions.nextLink && startTime + timeoutMs > Date.now());
 
-    cache.picks = cache.picks.concat(createQuickPickFromJsons(gitHubRepos, 'name', undefined, ['url', 'html_url']));
+    cache.picks = cache.picks.concat(createQuickPickFromJsons(gitHubRepos, 'name', undefined, ['url', 'html_url', 'name']));
```

Comment:
Can you remove `['url', 'html_url', 'name']` as a parameter? It should just be returning the original object in `data`...

The signature should be:
```
function createQuickPickFromJsons<T extends Object>(jsons: T[], label: string, description?: string): IAzureQuickPickItem<T>[] {
```